src/Commands/core/WatchdogCommands.php

```diff
@@ -47,6 +47,7 @@ public function __construct(protected Connection $connection)
     #[CLI\Option(name: 'severity-min', description: 'Restrict to messages of a given severity level and higher.')]
     #[CLI\Option(name: 'type', description: 'Restrict to messages of a given type.')]
     #[CLI\Option(name: 'extended', description: 'Return extended information about each message.')]
+    #[CLI\Option(name: 'date_format', description: 'Specify a date format for the date console output.')]
```

Comment:
We typically use dashes in option names not underscores.

Reply:
Fixed